miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -75,20 +78,36 @@ class MiniappSdkInitializer : ContentProvider() {
 
         MiniApp.init(
             context = context,
-            miniAppSdkConfig = MiniAppSdkConfig(
-                baseUrl = manifestConfig.baseUrl(),
-                rasProjectId = backwardCompatibleHostId,
-                rasAppId = manifestConfig.rasAppId(),
-                subscriptionKey = manifestConfig.subscriptionKey(),
-                hostAppUserAgentInfo = manifestConfig.hostAppUserAgentInfo(),
-                isPreviewMode = manifestConfig.isPreviewMode(),
-                isTestMode = manifestConfig.isTestMode()
-            )
+            miniAppSdkConfig = createMiniAppSdkConfig(manifestConfig, backwardCompatibleHostId)
         )
 
+        executeMiniAppAnalytics(backwardCompatibleHostId)
+
         return true
     }
 
+    private fun createMiniAppSdkConfig(
+        manifestConfig: AppManifestConfig,
+        backwardCompatibleHostId: String
+    ): MiniAppSdkConfig = MiniAppSdkConfig(
+        baseUrl = manifestConfig.baseUrl(),
+        rasProjectId = backwardCompatibleHostId,
+        rasAppId = manifestConfig.rasAppId(),
+        subscriptionKey = manifestConfig.subscriptionKey(),
+        hostAppUserAgentInfo = manifestConfig.hostAppUserAgentInfo(),
+        isPreviewMode = manifestConfig.isPreviewMode(),
+        isTestMode = manifestConfig.isTestMode()
+    )
+
+    private fun executeMiniAppAnalytics(backwardCompatibleHostId: String) {
+        MiniAppAnalytics.init(rasProjectId = backwardCompatibleHostId)
+        MiniAppAnalytics.instance?.sendAnalytics(
```

Comment:
It's a good practice to write note of analytics intention for future maintenance.

e.g. `// send analytics tracking when Host App is launched which contains our SDK.`

In other places:
`// send analytics tracking when Host App displays a mini app.`
`// send analytics tracking when Host App closes a mini app.`

Reply:
good idea.